modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -242,4 +250,22 @@ boolean isActiveActor() {
     public PlacementDriver placementDriver() {
         return leaseTracker;
     }
+
+    private void recoveryInternalComponents() {
```

Comment:
Should be called `recoverInternalComponents`

Reply:
fix it